jobs/aws/eks-distro-build-tooling/aws-for-fluent-bit-helm-chart-postsubmits.yaml

```diff
@@ -49,12 +49,6 @@ postsubmits:
             - -c
             - date +%s > /status/pending
           periodSeconds: 10
-      - name: buildkitd
-        image: moby/buildkit:master-rootless
-        command:
-        - sh
-        args:
-        - /script/entrypoint.sh
         livenessProbe:
```

Comment:
Livenessprobe configuration is attached to a specific container. So since we are removing the buildkit container entirely, retaining this livenessprobe config is not necessary.

Also more importantly livenessprobe is necessary only on jobs which build images and upload to ecr, so for jobs which don't do that, we don't need to add livenessprobe section at all.